package.json

```diff
@@ -8,7 +8,7 @@
   "version": "2.10.15",
   "private": true,
   "scripts": {
-    "postinstall": "bower install",
+    "postinstall": "cd public; npm install",
     "start": "node server.js",
```

Comment:
I tried `cd public; yarn` but got a consistent error, something related to package-lock.json.
anyway this works.

Reply:
Why `cd public` before `npm install`?
I don't see a package.json in `\public`

Reply:
this PR will add a public/package.json which should include only client packages.
(look ~3 file down / search this page for public/package.json)

Reply:
Ok, got it. Thanks.